### PR TITLE
LPS-93514 com.liferay.portal.util tests

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:backend-hypersonic

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplCanonicalURLTest.java [hunk 1]
```diff
@@ -124,10 +125,15 @@ public void setUp() throws Exception {
 		_layout2 = LayoutTestUtil.addLayout(
 			_group.getGroupId(), false, nameMap, friendlyURLMap);
 
+		String groupKey = PropsValues.VIRTUAL_HOSTS_DEFAULT_SITE_NAME;
```
Comment (tinatian) [inline modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplCanonicalURLTest.java:128]: @dantewang 
Just curious, why this is not a problem before?

Tina

**Comment by tinatian** [comment]
See https://github.com/shuyangzhou/liferay-portal/pull/8064

**Pull request closed without merging**